sip-servlets-test-suite/testsuite/src/test/java/org/mobicents/servlet/sip/testsuite/simple/ShootmeSipServletTest.java

```diff
@@ -844,6 +844,30 @@ public void testShootmeSendBye407Terminated() throws InterruptedException, SipEx
         assertEquals("CONFIRMED", allMessagesContent.get(0));        
     }
 	
+	public void testErrorResponseCodeForNonExistingDialog() throws InterruptedException, SipException, ParseException, InvalidArgumentException {
+        String fromName = "sender";
+        String fromSipAddress = "sip-servlets.com";
+        SipURI fromAddress = senderProtocolObjects.addressFactory.createSipURI(
+                fromName, fromSipAddress);
+                
+        String toUser = "receiverWithTag";
+        String toSipAddress = "sip-servlets.com";
+        SipURI toAddress = senderProtocolObjects.addressFactory.createSipURI(
+                toUser, toSipAddress);
+        
+        sender.setSendBye(false);
+        // This trick is for creating fake dialog at UA for sending sub sequence 
+        sender.sendSipRequest("INVITE", fromAddress, toAddress, null, null, false);
+        Thread.sleep(2000);
+        sender.sendInDialogSipRequest("INFO", "1", "text", "plain", null, null);
+        Thread.sleep(100);
+        assertTrue(sender.getFinalResponse().getStatusCode() == 481);

```

Comment:
I'm expecting 481 Call/Transaction does not exist respond to INFO request. it's not for initial INVITE.